cli/main.rs

```diff
@@ -0,0 +1,39 @@
+use std::env;
+
+use clap::Parser;
+use sp_runtime::{generic, traits::BlakeTwo256};
+use subxt::PolkadotConfig;
+use try_runtime_core::commands::TryRuntime;
+
+#[subxt::subxt(
+    runtime_metadata_url = "wss://rpc.polkadot.io:443",
```

Comment:
Thanks for the suggestions. I will take care of CI pipelines later on. Combining this comment with the other, I think the best solution will be to have:
 - generated polkadot runtime file for default, offline compilation
 - `build.rs` module for customizing runtime (however it will require network access or a custom file)